addons/website/static/src/snippets/s_countdown/000.js

```diff
@@ -44,6 +44,8 @@ const CountdownWidget = publicWidget.Widget.extend({
         this._initTimeDiff();
 
         this._render();
+        // removes the placeholder image used for preview when dragging around the snippet
+        [...document.querySelectorAll('img.s_preview:not(:only-child)')].forEach(x => x.parentNode.removeChild(x));
```

Comment:
document should be replaced by `this,el` or `this,$el` to scope the querySelection to the actual snippet